angular-ui/src/app/sidebar/recommendations/recommendations.component.ts

```diff
@@ -27,9 +27,9 @@ export class RecommendationsComponent implements OnInit {
   movies: Movie[] = [];
   recommendedMovies: Movie[] = [];
   apiKeyService: any;
-  apiUrl: any;
+  apiUrl: string = 'https://api.themoviedb.org/3/';
 
-  constructor(private http: HttpClient) {}
+  constructor(private http: HttpClient, private apiKeyService: ApikeyService) {}
 
```

Comment:
make sure ApikeyService is in your imports list at the top of the file.

Reply:
Moved it